client/components/image/index.tsx

```diff
@@ -0,0 +1,20 @@
+import classnames from 'classnames';
+import { SyntheticEvent } from 'react';
+
+const hideImageOnError = ( event: SyntheticEvent< HTMLImageElement > ) => {
```

Comment:
This seems to hide the site icon when the site icon loads fine:

<img width="1124" alt="image" src="https://user-images.githubusercontent.com/36432/183507967-705593ac-05fc-4995-a41c-9c2e170e53fc.png">

Is this intended?